HLTrigger/HLTcore/src/HLTPrescaleProvider.cc

```diff
@@ -299,13 +299,13 @@ std::pair<std::vector<std::pair<std::string, int> >, int> HLTPrescaleProvider::p
       GlobalLogicParser l1tGlobalLogicParser = GlobalLogicParser(l1tname);
       const std::vector<GlobalLogicParser::OperandToken> l1tSeeds = l1tGlobalLogicParser.expressionSeedsOperandList();
       int l1error(0);
-      int l1tPrescale(-1);
+      double l1tPrescale(-1);
       for (auto const& i : l1tSeeds) {
         const string& l1tSeed = i.tokenName;
         if (!l1tGlobalUtil_->getPrescaleByName(l1tSeed, l1tPrescale)) {
           l1error += 1;
         }
-        result.first.push_back(std::pair<std::string, int>(l1tSeed, l1tPrescale));
+        result.first.push_back(std::pair<std::string, double>(l1tSeed, l1tPrescale));
```

Comment:
Variable ```result``` would need its type changed, thus also requiring a change in return type of the method, std::pair<std::vector<std::pair<std::string, int> >, int> (changing the first int to a double). But again that will create problems with all code using this.